src/test/java/rx/swing/sources/ListSelectionEventSourceTest.java

```diff
@@ -0,0 +1,238 @@
+/**
```

Comment:
In general, let's file separate PRs for general cleanup. That said, thanks for spotting these, and I'm ok with including them this time.